tests/rt.conf

```diff
@@ -72,13 +72,13 @@ COMPILE | s2swa_faster | intel | -DAPP=S2SWA -DCCPP_SUITES=FV3_GFS_v17_coupled_p
 RUN | cpld_control_p8_faster                            | - cheyenne  noaacloud                 | baseline |
 
 # Unstructured WW3 mesh
-COMPILE | s2sw_pdlib | intel | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 -DPDLIB=ON | + hera orion cheyenne wcoss2 | fv3 |
-RUN | cpld_control_pdlib_p8                             | + hera orion cheyenne wcoss2    | baseline |
-RUN | cpld_restart_pdlib_p8                             | + hera orion cheyenne wcoss2    |          | cpld_control_pdlib_p8
-RUN | cpld_mpi_pdlib_p8                                 | + hera orion cheyenne wcoss2    |          | cpld_control_pdlib_p8
+COMPILE | s2sw_pdlib | intel | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 -DPDLIB=ON | - noaacloud | fv3 |
+RUN | cpld_control_pdlib_p8                             | - noaacloud                | baseline |
```

Comment:
@DeniseWorthen can you chime in on this question, too?   There are a few differences between the pdlib tests and the gfs tests such as 64 vs 32 bit and inner vs outer loop coupling.  For GFS, the needs are met with the gfs tests, but there might be other work I'm not aware of which necessitates the pdlib tests in addition to the gfs tests.

Reply:
I think this is a question you could ask about almost any test in the rt.conf. Is it targeting an operational implementation? If we're not going to be running WW3 on a structured grid for ops, then why maintain all the tests which currently use a structured ww3 mesh (ie, cpld_control_p8 etc)? It's a question which has dogged the RT system for much longer than just this PR. 

My personal preference would be to maintain pdlib tests which do *not* use the outer loop, since I suspect none of the potential coastal applications or perhaps even at some point the HAFS application will not use unstructured in the outer loop.

Reply:
One quick FYI: GEFS will  continue to use the structured grid in operations (although in the outerloop) and currently the closest test to GEFS for now is cpld_control_p8.   @BinLiu-NOAA can confirm, but I believe HAFSv2 will continue to persist with a structured grid, although I'm sure it'll examine the possibility of unstructured grids in the future and hopefully innerloop coupling with waves will be technically more realistic at that point.

Reply:
Wave-ice coupling does require both inner-loop coupling as well as unstructured mesh, so I'll create an issue for converting the existing pdlib tests to turn on wave-ice coupling.